### ISSUE TITLE
builtin print function not found when called inside a module

### ISSUE THREAD

**Issue by 0verse** [mini-rose/coffee#12]
I have a module named strinc.ff which cantains:
```
fn print_str(s: str) {
    print(s)
}
```
The hello.ff file contains:
```
use "strinc"

fn main() {
    _str: str = "hello world"
    print_str(_str)
}
```
I get this error message while compiling:

> özcü@özcül ~/çizeylemcil/coffee # nxg -M -o hello hello.ff 
> 
> error in strinc.ff:
> 
> 1	    print(s)
> 	             ^~~~~ no function named `print` found
>

**Comment by bellrise** [comment]
I know what the problem is, I should find some time tommorow to fix it.

**Comment by bellrise** [comment]
Fixed as of e0affe7.

**Comment by 0verse** [comment]
thanks a lot :)
